src/Directory.Build.props

```diff
@@ -63,6 +63,7 @@
 
   <ItemGroup>
     <PackageReference Condition="'$(DisableNerdbankVersioning)' != 'true'" Include="Nerdbank.GitVersioning" Version="$(GitVersioningVersion)" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0-alpha-003" PrivateAssets="all" Condition="'$(OsEnvironment)'!='windows'"/>
```

Comment:
Wouldn't we want this on Windows as well? To unify logic? Or is there a reason to keep things the way they are?

Reply:
I was getting a bunch of errors along the lines of
```
ManifestUtil\TrustInfo.cs(338,17): error CS0246: The type or namespace name 'PermissionSet' could not be found (are you missing a using directive or an assembly reference?) [D:\j\w\innerloop_Win---c5dcd6be\src\Tasks\Microsoft.Build.Tasks.csproj]
NativeMethods.cs(25,6): error CS0246: The type or namespace name 'GuidAttribute' could not be found (are you missing a using directive or an assembly reference?) [D:\j\w\innerloop_Win---c5dcd6be\src\Tasks\Microsoft.Build.Tasks.csproj]
ManifestUtil\TrustInfo.cs(396,16): error CS0518: Predefined type 'System.Object' is not defined or imported [D:\j\w\innerloop_Win---c5dcd6be\src\Tasks\Microsoft.Build.Tasks.csproj]
```

So it seemed like I should keep the Windows build the way it was.  Any idea on those errors?

Reply:
Also cc @weshaggard if the errors look familiar.